table/plain_table_reader.cc

```diff
@@ -308,7 +308,7 @@ Status PlainTableReader::PopulateIndex(TableProperties* props,
   index_in_file &= s.ok();
```

Comment:
So now index_in_file will be false even if the index block is in file but the bloom block is not ?
Does that mean that we don't get any benefit from storing the index in file ?